.gitignore

```diff
@@ -1,27 +1 @@
-# Build
-lib/*
-
-###Node###
-
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Dependency directory
-# Commenting this out is preferred by some people, see
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
 node_modules
-
-# Users Environment Variables
-.lock-wscript
```

Comment:
@nono Yeah i know, i was just wondering if it wasn't more safe to add those in the gitignore of the project. But i was really just wondering, i'm not gonna fight for this :)